json_feature.cpp

```diff
@@ -37,4 +48,3 @@ void JSONFeature::append_to(std::string& buffer) {
     buffer.append(m_stream.GetString(), m_stream.GetSize());
     buffer.append(1, '\n');
 }
-
```

Comment:
Try to not make any extra whitespace changes or so that have nothing to do with the issue at hand.